reporter/html.js

```diff
@@ -533,7 +533,8 @@ function appendTest( name, testId, moduleName ) {
 
 // HTML Reporter initialization and load
 QUnit.begin(function( details ) {
-	var qunit = id( "qunit" );
+	var qunit = id( "qunit" ),
+		tests;
```

Comment:
Will update. I don't see that particular rule mentioned in the [jQuery JS style guide](https://contribute.jquery.org/style-guide/js/) though, or perhaps I need to look elsewhere? Regardless, should probably get #936 merged so that it's easier to pick up on these before PR.